components/landing/challenge/challenge-intro.tsx

```diff
@@ -0,0 +1,75 @@
+import { Badge } from "@/components/ui/badge";
+import Link from "next/link";
+import {
+    Card,
+    CardContent,
+    CardDescription,
+    CardHeader,
+    CardTitle,
+    CardFooter,
+} from "@/components/ui/card";
+import { Separator } from "@/components/ui/separator";
+
+interface ChallengeIntroProps {
+    button: () => React.ReactNode;
+}
+
+const ChallengeIntro: React.FC<ChallengeIntroProps> = ({ button }) => {
+    return (
+        <Card className='mx-auto my-8 h-[420px] w-2/3 overflow-hidden rounded-2xl bg-gray-800'>
+            <CardHeader className='p-4'>
+                <CardTitle className='mb-4 text-3xl font-semibold text-white'>
+                    🥳 We&apos;re glad you&apos;re here!
+                </CardTitle>
+                <CardDescription>
+                    <Badge className='mx-1 rounded-full bg-blue-400 px-2 py-1 text-white'>
+                        Alpha
```

Comment:
I think all of these alpha badges are just white noise,  With the timeline and other references to the fact that we're still in development, these badges are obviously just added to try and add some decoration, but it's kind of just adding stuff for the sake of adding stuff.